stdlib/ballerina-http/src/main/ballerina/http/Package.md

```diff
@@ -36,6 +36,43 @@ See [Mutual SSL Example](https://ballerinalang.org/docs/by-example/mutual-ssl).
 
 See [Caching Example](https://ballerinalang.org/docs/by-example/caching), [HTTP Disable Chunking Example](https://ballerinalang.org/docs/by-example/http-disable-chunking).
 
+###WebSockets
+The package also provides support for WebSockets. There are two types of WebSocket endpoint which are `WebSocketClient` and `WebSocketListener`. Both endpoints support all WebSocket frames. The `WebSocketClient` has a callback service.
+
+There are also two types of services in WebSocket which are `WebSocketService` and `WebSocketClientService`. The callback service for `WebSocketClient` would always be a `WebSocketClientService`. The WebSocket services have a fixed set of resources which do not have a resource config. The incoming messages are passed to these resources.
```

Comment:
Same construction here: "There are also two types of services in WebSocket: `WebSocketService` and `WebSocketClientService`."

Reply:
Change "would always be" to "is always"

Reply:
Change "resources which" to "resources that"